getting-started/introduction-to-aepedge.md

```diff
@@ -6,27 +6,29 @@ This documentation section is in the process of being moved - please see [https:
 
 Adobe Experience Cloud implementations for mobile apps leverage [Adobe Experience Platform Mobile SDKs](http://sdkdocs.com) for data collection and experience delivery. With the introduction of Adobe Experience Platform Edge Network data collection, client-side implementations will provide unified, purpose-based APIs to provide customers with a more modern and efficient data collection paradigm.
 
-By utilizing the latest Adobe Experience Platform Mobile SDK along with the [Adobe Experience Platform Edge Network extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/415ad4c45b0bddcb6879a7403d8c2922f154ef8b/getting-started/using-mobile-extensions/adobe-edge/README.md), customers can adopt XDM \([Experience Data Model](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html)\) workflows for data collection. XDM data collection represents a shift from solution specific APIs to simple key / value pairs, that can be sent from both Mobile and Web properties.
+By utilizing the latest Adobe Experience Platform Mobile SDK along with the [Adobe Experience Platform Edge Network extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/415ad4c45b0bddcb6879a7403d8c2922f154ef8b/getting-started/using-mobile-extensions/adobe-edge/README.md), customers can adopt [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html) workflows for data collection. XDM data collection represents a shift from solution specific APIs to simple key value pairings, that can be sent from both Mobile and Web properties.
 
-For instance, XDM events based on take the place of solution APIs such as trackAction and trackState calls to send analytics events. Customers that are not yet ready to fully adopt an XDM only data workflow may opt to keep solution specific API calls, and then create separate XDM calls for important events.
+For instance, XDM events based on ??? can take the place of solution APIs such as trackAction and trackState calls to send analytics events. Customers that are not yet ready to fully adopt an XDM only data workflow may opt to keep solution specific API calls, and then create separate XDM calls for important events.
```

Comment:
"???" here on line 11 seems like a typo, or is it something to be filled in later on?

Reply:
We can probably remove the first sentence altogether as it doesn't make sense in this context - this one:
`For instance, XDM events based on ??? can take the place of solution APIs such as trackAction and trackState calls to send analytics events.`